src/components/shared/DataTypeSelect.vue

```diff
@@ -29,6 +29,10 @@ export default {
       type: String,
       default: undefined,
     },
+    allowNone: {
+      type: Boolean,
+      default: true,
```

Comment:
I would set this to false so as to not change the existing implementation

Reply:
I have changed it but setting this to true or false means no tests fail. Is there a missing test somewhere?

Reply:
Yes probably. LibraryTypeSelect has the same logic and it has a test here: https://github.com/sanger/traction-ui/blob/develop/tests/unit/components/shared/LibraryTypeSelect.spec.js#L24